ModelFarm/ModelFarm.gradle

```diff
@@ -11,7 +11,7 @@ configurations {
 }
 
 dependencies {
-    compile project(':engine-table')
+    implementation project(':engine-table')
```

Comment:
Yep, and should have api on engine-time after digging in.

Reply:
`engine-api` declares an `api` dependency on `engine-time`, so we don't *need* to add it directly, but it's not harmful.

Reply:
It's (usually) best-practice to explicitly declare everything you are re-exposing, even if you would otherwise inherit it IMO. It's more important with external dependencies (b/c a version bump may change what they expose).